db/seeds.rb

```diff
@@ -13,7 +13,7 @@
 Video.create(title: 'South Park', description: 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut',
              avatar: 'south_park.jpg', category: comedies)
 
-Video.create(title: 'Monk', description: 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut',
+monk = Video.create(title: 'Monk', description: 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut',
```

Comment:
I would format this differently.

```ruby
monk = Video.create!(title: 'Monk',
                     description: Faker::Lorem.paragraph,
                     avatar: 'monk.jpg',
                     img: 'monk_large.jpg',
                     category: dramas)
```
Also consider using faker with random text.